packages/create/templates/hello/package.json

```diff
@@ -3,25 +3,25 @@
   "version": "0.0.1",
   "scripts": {
     "build": "tsc",
-    "test": "npx knex migrate:rollback && npx knex migrate:up && jest",
+    "test": "npx dbos rollback && npx dbos migrate && jest",
     "lint": "eslint src",
     "lint-fix": "eslint --fix src",
     "dev": "nodemon",
-    "start": "npx dbos-sdk start"
+    "start": "npx dbos start"
   },
   "devDependencies": {
     "@dbos-inc/eslint-plugin": "^0.0.6",
-    "@types/jest": "^29.5.5",
-    "@types/supertest": "^2.0.14",
-    "eslint": "^8.56.0",
+    "@types/jest": "^29.5.12",
+    "@types/supertest": "^2.0.16",
+    "eslint": "^8.57.0",
     "jest": "^29.7.0",
     "nodemon": "^3.1.0",
     "supertest": "^7.0.0",
-    "ts-jest": "^29.1.1",
-    "typescript": "^5.2.2"
+    "ts-jest": "^29.1.2",
```

Comment:
Is OpenAPI supporting TS 5.4 now?

Reply:
This is for the template--it's safe to update independently of the main package (it's actually always been on 5.4, this just made it explicit).